### PR TITLE
[feature] Instance Resize

### PR DESCRIPTION
**Pull request by SunithaGudisagarIBM**
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/2338

https://jiracloud.swg.usma.ibm.com:8443/browse/UI-14664
Screenshot 2021-03-11 at 1 37 49 PM

### CODE REVIEW

**Comment by kavya498** [comment]
@SunithaGudisagarIBM 
Please resolve the conflicts, eliminate false positive secrets and update the pr..

**Comment by SunithaGudisagarIBM** [comment]
@hkantare Can you please review this PR

File: ibm/resource_ibm_is_instance.go [hunk 1]
```diff
@@ -1856,6 +1856,49 @@ func instanceUpdate(d *schema.ResourceData, meta interface{}) error {
...
+		instancePatchModel := &vpcv1.InstancePatch{
+			Profile: profile,
+		}
+		instancePatch, err := instancePatchModel.AsPatch()
+		if err != nil {
+			return fmt.Errorf("Error calling asPatch for InstancePatch: %s", err)
+		}
+		updnetoptions.InstancePatch = instancePatch
+
+		_, _, err = instanceC.UpdateInstance(updnetoptions)
```
Comment (hkantare) [inline ibm/resource_ibm_is_instance.go:1895]: please catch the response and return the response along with error if any error

File: ibm/resource_ibm_is_instance.go [hunk 2]
```diff
@@ -1856,6 +1856,49 @@ func instanceUpdate(d *schema.ResourceData, meta interface{}) error {
...
+			Type:       &actiontype,
+		}
+		_, response, err := instanceC.CreateInstanceAction(createinsactoptions)
+		if err != nil {
+			if response != nil && response.StatusCode == 404 {
+				return nil
+			}
+			return fmt.Errorf("Error Creating Instance Action: %s\n%s", err, response)
+		}
+		_, err = isWaitForInstanceActionStop(instanceC, d.Timeout(schema.TimeoutDelete), id, d)
```
Comment (hkantare) [inline ibm/resource_ibm_is_instance.go:1873]: use schema.TimeoutUpdate here since its an uodate operation

**Pull request closed without merging**